ramp-frontend/ramp_frontend/views/general.py

```diff
@@ -11,7 +12,12 @@
 @mod.route('/')
 def index():
     """Default landing page."""
-    return render_template('index.html')
+    img_ext = ('.png', '.jpg', '.jpeg', '.gif', '.svg')
+    images = [f for f in os.listdir(
+              'ramp-frontend/ramp_frontend/static/img/powered_by/')
```

Comment:
This path should be with respect to `__file__`, otherwise it does not exist https://github.com/paris-saclay-cds/ramp-board/issues/300